packages/api/src/controllers/v2/community/details.ts

```diff
@@ -16,30 +16,18 @@ class CommunityController {
     getManagers = (req: RequestWithUser, res: Response) => {
```

Comment:
We can do something like https://github.com/impactMarket/backend/blob/main/packages/api/src/controllers/v2/microcredit/list.ts#L22, so we avoid the casts like `orderBy as string` or  `lastActivity_lt as string` below.

Reply:
done!